<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserGUIClass</class>
 <widget class="QWidget" name="UserGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserGUI</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>160</width>
     <height>176</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="adoptButton">
      <property name="text">
       <string>Adopt</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="filterButton">
      <property name="text">
       <string>Filter by age and breed</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="adoptionButton">
      <property name="text">
       <string>Adoption List</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exportButton">
      <property name="text">
       <string>Export Adoption List</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exitButton">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="dataTableView">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>21</y>
     <width>431</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
